daemon/daemon_unix.go

```diff
@@ -1269,12 +1269,17 @@ func (daemon *Daemon) initCgroupsPath(path string) error {
 	// for the period and runtime as this limits what the children can be set to.
 	daemon.initCgroupsPath(filepath.Dir(path))
 
-	_, root, err := cgroups.FindCgroupMountpointAndRoot("cpu")
+	mnt, root, err := cgroups.FindCgroupMountpointAndRoot("cpu")
 	if err != nil {
 		return err
 	}
+	// When docker is run inside docker, the root is based of the host cgroup.
+	// Should this be handled in runc/libcontainer/cgroups ?
```

Comment:
I don't think this can be handled by `runc`. This is basically the `cgroupsPath` that is passed to `runc` within the `config.json`

e.g.
```
"cgroupsPath": "/docker/932f0ceec93d425a5e3ea0ce85c7323388ae9f4d9ae80f18b0aa39c20849f30f"
```

The mountinfo simply reflects that, which is correct.